### PR TITLE
feat: Add support for email_address & phone_number CRUD operations

### PR DESCRIPTION
**Pull request by yourtallness**
## Type of change

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [X] 🌟 New feature (non-breaking change which adds functionality)
- [ ] 🔨 Breaking change (fix or feature that would cause existing functionality)
- [ ] 📖 Docs change / refactoring / dependency upgrade to change)

## Description

Adds support for the following `email_address` operations:

* Create
* Read
* Update
* Delete

Adds support for the following `phone_number` operations:

* Create
* Read
* Update
* Delete

### CODE REVIEW

**Review by gkats — CHANGES_REQUESTED** [review]

File: tests/integration/email_addresses_test.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	users, _ := client.Users().ListAll(clerk.ListAllUsersParams{})
+	if users == nil || len(users) == 0 {
+		return
+	}
+
+	user := users[0]
+
+	// create
+
+	fakeEmailAddress := gofakeit.Email()
```
[the remark this answers is not in the document]
Comment (yourtallness) [reply tests/integration/email_addresses_test.go:26]: I prefer this to creating random emails / phones myself tbh

[the remark this answers is not in the document]
Comment (gkats) [reply tests/integration/email_addresses_test.go:26]: I think it's not a matter of preference, but constraints. This is a package that will be downloaded and installed by others, so it's important to keep dependencies to a minimum.

I don't think that test dependencies are excluded in Go, right?

[the remark this answers is not in the document]
Comment (yourtallness) [reply tests/integration/email_addresses_test.go:26]: Test dependencies are not included in the build, so we should be OK.

**Review by agis — APPROVED** [review]
LGTM!

**Pull request merged**